flight/states/odlcs.py

```diff
@@ -29,4 +32,28 @@ async def run(self, drone: System) -> AirDrop:
         AirDrop : State
             The next state in the state machine, AirDrop for the payloads
         """
+
+        # Camera gets turned on
+        await drone.camera.set_mode(Mode.PHOTO)
+        logging.info("Camera changed to Photo mode")
+
+        # These waypoint values are all that are needed to traverse the whole odlc drop location because it is a small rectangle
+        # The first waypoint is the midpoint of the left side of the rectangle(one of the short sides), the second point is the
+        # midpoint of the right side of the rectangle(other short side), and the third point is the top left corner of the rectangle
+        # it goes there for knowing where the drone ends to travel to each of the drop locations, the altitude is locked at 85 because
+        # we want the drone to stay level and the camera to view the whole odlc boundary
+        waypoint: dict[str, list[float]] = {
+            "lats": [38.31451966813249, 38.31430872867596, 38.31461622313521],
+            "longs": [-76.54519982319357, -76.54397320409971, -76.54516993186949],
+            "Altitude": [85],
+        }
+
+        # traverses the 3 waypoints starting at the midpoint on left to midpoint on the right then to the top left corner at the rectangle
+        point: int
+        for point in range(3):
+            logging.info("Moving")
```

Comment:
Since this is the ODLC movement logic for the actual competition, this logging statement should probably be a bit more descriptive, like "Begin ODLC pathing" or state which section of the rectangle the drone is currently in.